packages/app/src/components/nominationBanner/style.css

```diff
@@ -17,7 +17,20 @@
 }
 
 .decline-button {
+  background-color: #f72314;
+  color: #ffffff;
+  border: #8a8a8a;
+  font-size: 0.82rem;
```

Comment:
Why `0.82rem` here? Seems oddly specific, given most other buttons appear to have a font size of `1rem`.

Reply:
I corrected this. I created a standard .button under App.css so all future buttons should share the same styling.